configure.in

```diff
@@ -1345,10 +1346,8 @@ version or omitting it entirely."
             #elif !defined(__IPHONE_OS_VERSION_MIN_REQUIRED)
             #error unrecognized platform
             #endif
-        ],
-        [AC_MSG_RESULT([yes])],
-        [AC_MSG_FAILURE([$error_message])]
-    )
+        ]])],[AC_MSG_RESULT([yes])],[AC_MSG_FAILURE([$error_message])
```

Comment:
I'd really appreciate if you could please restore the previous indentation which is much more readable, as otherwise I'd have to do it myself.

(this applies here and in several more places below)

Reply:
Will do!